recipes/open62541/all/conanfile.py

```diff
@@ -0,0 +1,238 @@
+from conans.model.version import Version
+from conans.errors import ConanInvalidConfiguration
+from urllib.parse import urlparse
+import shutil
+
+
+class Open62541Conan(ConanFile):
+    name = "open62541"
+    license = "MPLv2"
+    exports_sources = [
+        "CMakeLists.txt",
+        "patches/**"
+    ]
+    homepage = "https://open62541.org/"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "open62541 is an open source and free implementation of OPC UA (OPC Unified Architecture) written in the common subset of the C99 and C++98 languages. The library is usable with all major compilers and provides the necessary tools to implement dedicated OPC UA clients and servers, or to integrate OPC UA-based communication into existing applications. open62541 library is platform independent. All platform-specific functionality is implemented via exchangeable plugins. Plugin implementations are provided for the major operating systems."
+    topics = (
+        "OPC UA", "open62541", "sdk", "server/client", "c", "iec-62541",
+        "industrial automation", "tsn", "time sensetive networks", "publish-subscirbe", "pubsub"
+    )
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False],
+        "historize": [True, False],
+        "logging_level": ["Fatal", "Error", "Warning", "Info", "Debug", "Trace"],
+        "subscription": [True, False],
+        "subscription_events": [True, False],
+        "methods": [True, False],
+        "dynamic_nodes": [True, False],
+        "single_header": [True, False],
+        "multithreading": [True, False],
+        "imutable_nodes": [True, False],
+        "web_socket": [True, False],
+        "discovery": [True, False],
+        "discovery_semaphore": [True, False],
+        "discovery_multicast": [True, False],
+        "query": [True, False],
+        "encription": [True, False],
+        "json_support": [True, False],
+        "pub_sub": ["None", "Simple", "Ethernet", "Ethernet_XDP"],
+        "data_access": [True, False],
+        "compiled_nodeset_descriptions": [True, False],
+        "namescpae_zero": ["MINIMAL", "REDUCED", "FULL"],
+        "embedded_profile": [True, False],
+        "typenames": [True, False],
+        "hardening": [True, False],
+        "cpp_compatible": [True, False],
+        "readable_statuscodes": [True, False]
+    }
+    default_options = {
+        "fPIC": True,
+        "shared": True,
+        "historize": False,
+        "logging_level": "Info",
+        "subscription": True,
+        "subscription_events": False,
+        "methods": True,
+        "dynamic_nodes": True,
+        "single_header": False,
+        "multithreading": False,
+        "imutable_nodes": False,
+        "web_socket": False,
+        "discovery": True,
+        "discovery_semaphore": True,
+        "discovery_multicast": False,
+        "query": False,
+        "encription": False,
```

Comment:
This should be encryption

Reply:
Resolved in the next commit, just running local tests